gapis/api/vulkan/read_framebuffer.go

```diff
@@ -807,57 +904,13 @@ func postImageData(ctx context.Context,
 				VkResult_VK_SUCCESS,
 			),
 		)
+	} else {
+		resolveImageID = VkImage(0)
+		resolveImageMemoryID = VkDeviceMemory(0)
 	}
 
-	// Create command pool, allocate command buffer
-	writeEach(ctx, out,
-		cb.VkCreateCommandPool(
-			vkDevice,
-			commandPoolCreateInfoData.Ptr(),
-			memory.Nullptr,
-			commandPoolData.Ptr(),
-			VkResult_VK_SUCCESS,
-		).AddRead(
-			commandPoolCreateInfoData.Data(),
-		).AddWrite(
-			commandPoolData.Data(),
-		),
-		cb.VkAllocateCommandBuffers(
-			vkDevice,
-			commandBufferAllocateInfoData.Ptr(),
-			commandBufferData.Ptr(),
-			VkResult_VK_SUCCESS,
-		).AddRead(
-			commandBufferAllocateInfoData.Data(),
-		).AddWrite(
-			commandBufferData.Data(),
-		),
-	)
-
-	// Create a fence
-	writeEach(ctx, out,
-		cb.VkCreateFence(
-			vkDevice,
-			fenceCreateData.Ptr(),
-			memory.Nullptr,
-			fenceData.Ptr(),
-			VkResult_VK_SUCCESS,
-		).AddRead(
-			fenceCreateData.Data(),
-		).AddWrite(
-			fenceData.Data(),
-		),
-	)
```

Comment:
So no need for a fence anymore?

Reply:
Nope, there was never really a need, we used vkDeviceWaitIdle anyway.